codegen/cmd/injection-gen/generators/reconciler_controller.go

```diff
@@ -154,7 +154,10 @@ const (
 	defaultControllerAgentName = "{{.type|lowercaseSingular}}-controller"
 	defaultFinalizerName       = "{{.type|allLowercasePlural}}.{{.group}}"
 	defaultQueueName           = "{{.type|allLowercasePlural}}"
-	{{if .hasClass}}classAnnotationKey = "{{ .class }}"{{end}}
+	{{if .hasClass}}
+	// ClassAnnotationKey points to the annotation for the class of this resource.
+	ClassAnnotationKey = "{{ .class }}"
```

Comment:
you want to export it?

Reply:
maybe what you are after is an accessor? like `GetClass` and `SetClass` implemented in the reconciler as exported functions? Then you never need to know what the class annotation key (or the details of how it is set). This would save us from rewriting a bunch of things if we move to an annotation or a spec field for some reason

Reply:
IIUC this is to make the comment the source of truth, which is something I was struggling with us splitting.  So 👍 from me 😉

Reply:
Sounds good, I can explore that direction!